### PR TITLE
Use explicit winnr with O mapping

### PR DESCRIPTION
**Pull request by blueyed**
With autocommands etc you might not end up in the previous/qf window
otherwise.

### CODE REVIEW

**Comment by blueyed** [comment]
This could also make use of the method in #46, but is more simple this way.

**Comment by mhinz** [comment]
Thanks, merged!

File: autoload/grepper.vim [hunk 1]
```diff
@@ -455,6 +455,7 @@ function! s:finish_up(flags) abort
     nnoremap <silent><buffer> <cr> <cr>zv
     nnoremap <silent><buffer> o    <cr>zv
     nnoremap <silent><buffer> O    <cr>zv<c-w>p
+    nnoremap <silent><buffer> O    :let w=winnr()<cr><cr>zv:exec w 'wincmd p'<cr>
```
Comment (blueyed) [inline autoload/grepper.vim]: Sorry, forgot to remove the original mapping: https://github.com/mhinz/vim-grepper/pull/49.

Comment (mhinz) [inline autoload/grepper.vim]: D'oh, I saw while I read the diff, but forgot to remove it myself..

Comment (blueyed) [inline autoload/grepper.vim]: Probably because you got distracted by renaming the var?! ;)

Comment (mhinz) [inline autoload/grepper.vim]: Hah, well, the more important thing was adding the `unlet` because global var and all that. :-P

**Pull request closed without merging**